### ISSUE TITLE
dsPIC30 ElfLoader Exception

### ISSUE THREAD

**Issue by ryanmkurtz** [NationalSecurityAgency/ghidra#1699]
```
#include <stdio.h>
#include <xc.h>

__prog__ const char __attribute__((space(psv))) hello_world[] = "Hello World";

int main(void) {

    printf("%s\n", hello_world);
    printf("Hello World\n");
}
```
[Test.X.production.zip](https://github.com/NationalSecurityAgency/ghidra/files/4416831/Test.X.production.zip)

The `ElfLoader` throws an exception during import and the binary fails to load.  If I remove the line `printf("Hello World\n");` it loads fine and the PSV reference looks correct.

**Comment by gtackett** [comment]
Something similar happened today for me the first time (ever) that I tried to load a coff .a library. Probably will follow up here tomorrow, as it seems easy to reproduce.

**Comment by ryanmkurtz** [comment]
Fixed by 305a1ddf98076cfe540e1d7046dc774fd6b89a67

**Comment by emteere** [comment]
Fixed in master and future 9.2
